beets/ui/commands.py

```diff
@@ -1398,6 +1398,16 @@ def show_version(lib, opts, args):
 version_cmd.func = show_version
 default_commands.append(version_cmd)
 
+# database_location: return true if user
+# wants to create the parent directories.
+
+
+def database_dir_creation(path):
+    # Ask the user for a choice.
+    return ui.input_yn("The database directory {} does not \
+                       exists, create it (Y/n)?"
```

Comment:
```suggestion
                       exists. Create it (Y/n)?"
```

So it's two full sentences.